pkgs/os-specific/linux/syslinux/default.nix

```diff
@@ -68,12 +79,16 @@ stdenv.mkDerivation {
   ];
 
   postPatch = ''
-    substituteInPlace Makefile --replace /bin/pwd $(type -P pwd)
-    substituteInPlace utils/ppmtolss16 --replace /usr/bin/perl $(type -P perl)
+    substituteInPlace Makefile --replace-fail /bin/pwd $(type -P pwd)
+    substituteInPlace utils/ppmtolss16 --replace-fail /usr/bin/perl $(type -P perl)
 
     # fix tests
     substituteInPlace tests/unittest/include/unittest/unittest.h \
-      --replace /usr/include/ ""
+      --replace-fail /usr/include/ ""
+
+    # needs to be manually put in place because submodule hash contains an extra g in the beginning gd34132e
```

Comment:
I'm not sure the `g` is the problem here: the error message in the current behavior has `fatal: Fetched in submodule path 'gnu-efi', but it did not contain d34132e62f666904158c7ec2f1eef5a9d5281c36. Direct fetching of that commit failed.`, so it seems to get the git hash just fine.

Reply:
Then my best next guess is, that sourceforge is weird. I didn't want to click through many pages to find the old release.